.github/workflows/unit-tests.yml

```diff
@@ -49,8 +46,15 @@ jobs:
 
   test-android:
     name: unit-test-android
-    runs-on: macos-13
+    runs-on: ubuntu-24.04
     steps:
+      - name: Enable KVM group perms
```

Comment:
Can you add more of a description here of what this is for? (e.g. allowing the Android Emulator to use KVM acceleration)